src/main/java/seedu/allonus/storage/StorageFile.java

```diff
@@ -130,6 +130,9 @@ public String getModuleInFileFormat(int listIndex) {
         case "Tutorial":
             category = "tut";
             break;
+        case "Laboratory":
```

Comment:
@OzairHasan I have introduced a new parameter for category. I have reflected this in storage file as well!